exasol/slc/internal/tasks/build/docker_flavor_image_task.py

```diff
@@ -120,13 +121,17 @@ def get_mapping_of_build_files_and_directories(self) -> Dict[str, str]:
         return result
 
     def get_build_step_path(self) -> Path:
-        return self.get_path_in_flavor_path() / self.get_build_step()
+        if build_step_path := self.get_build_step():
+            return self.get_path_in_flavor_path() / build_step_path
+        else:
+            return self.get_path_in_flavor_path()
 
     def get_path_in_flavor_path(self) -> Path:
+        flavor_path: str = self.flavor_path  # type: ignore
```

Comment:
Doesn't ITDE specify a type hint for `FlavorBaseTask.flavor_path` (I saw `luigi.Parameter()`)?

Reply:
Yes, the type hint in ITDE needs, to be fixed.
I opened https://github.com/exasol/integration-test-docker-environment/issues/445